BinaryObjectScanner/FileType/DiskImage.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using BinaryObjectScanner.Data;
+using BinaryObjectScanner.Interfaces;
+using SabreTools.IO.Extensions;
+using SabreTools.Serialization.Wrappers;
+
+namespace BinaryObjectScanner.FileType
+{
+    /// <summary>
+    /// .iso file
```

Comment:
Nitpick: This file isn't just for ISO anymore because you have ISO9660 specifically split out